### PR TITLE
Hide inactive associations from normal users

### LINKED ISSUE #78: Possibility to mark associations as inactive
Since Sirius is leaving SBZ, it is useful to be able to mark associations as 'inactive'

### PR DESCRIPTION
**Pull request by bramvandartel**
Fixes #78.

Only superusers and foundation managers will be able to see inactive users. Furthermore, weekly reminder emails will never be sent to inactive associations.

### CODE REVIEW

**Comment by WKobes** [comment]
@bramvandartel please resolve comments or close PR

**Comment by bramvandartel** [comment]
Which comment? The pipeline is dead so no pass on testing :(

**Review by WKobes — CHANGES_REQUESTED** [review]
Now I see the comments were never placed, but still in draft. my mistake

File: alexia/apps/organization/managers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    def filter_active(self, request):
+        """
+        Filters active organizations unless superuser or foundation manager
+        """
+        only_active = not (hasattr(request, 'user') and hasattr(request.user, 'profile') and (request.user.is_superuser or request.user.profile.is_foundation_manager))
+
+        if only_active:
+            return self.filter(is_active=True)
+        else:
+            return self.all()
```
Comment (WKobes) [inline alexia/apps/organization/managers.py:14]: newline

File: alexia/apps/scheduling/management/commands/remindermail.py [hunk 2]
```diff
@@ -12,12 +12,13 @@ def handle(self, *args, **options):
         for mailtemplate in MailTemplate.objects.filter(name=MailTemplate.REMINDER, is_active=True):
```
Comment (WKobes) [inline alexia/apps/scheduling/management/commands/remindermail.py:12]: better, but is it not possible to move the active check to this for loop? `organization__is_active=True` or smth?

**Review by WKobes — APPROVED** [review]

**Comment by WKobes** [comment]
Feel free to merge, good work

**Pull request merged**